File: C++/Problems/Problems/problem_3512.cpp
#include "problem_3512.h"

bool problem_3512::test() {
    std::vector nums { 3, 9, 7 };
    int k = 5;

    int expected = 4;

    auto result = minOperations(nums, k);

    std::cout << result << std::endl;

    return expected == result;
}

int problem_3512::minOperations(std::vector<int> &nums, int k) {
    int n = nums.size();
    int sum = 0;
    for (int i = 0; i < n; ++i)
    {
        sum += nums[i];
    }

    return sum % k;
}

File: Python/Problems/Problems/problem3512.py
import heapq
import math
import sys

import numpy as np

from utils import print1DArray, print2DArray
from typing import List, Set
from collections import Counter

from .problem import Problem


class Problem3512(Problem):
    def test(self) -> bool:
        nums = [3, 9, 7]
        k = 5

        expected = 4

        result = self.minOperations(nums, k)

        print(result)

        return expected == result

    def minOperations(self, nums: List[int], k: int) -> int:
        return sum(nums) % k